common/src/main/java/org/apache/falcon/update/UpdateHelper.java

```diff
@@ -46,13 +46,13 @@
 
     private static final String[] FEED_FIELDS = new String[]{"partitions", "groups", "lateArrival.cutOff",
                                                              "schema.location", "schema.provider", "tags",
-                                                             "group", "owner", "permission", };
```

Comment:
No. it checks at main class level. It filters out whole sla tag and ignores the changes in it.